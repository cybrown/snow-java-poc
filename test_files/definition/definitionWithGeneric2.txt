def a: List<List<int> > = aListFromSomewhere
