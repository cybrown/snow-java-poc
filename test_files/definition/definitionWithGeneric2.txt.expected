<Definition>
    <Identifier>a</Identifier>
    <Generic>
        <Identifier>List</Identifier>
        <Generic>
            <Identifier>List</Identifier>
            <Identifier>int</Identifier>
        </Generic>
    </Generic>
    <Identifier>aListFromSomewhere</Identifier>
</Definition>
